cross/dkb-trace/Makefile

```diff
@@ -0,0 +1,19 @@
+PKG_NAME = dkb-trace
```

Comment:
Why not use the maintained https://github.com/POV-Ray/povray? As far as I understand it is based from the dkb-trace source.

Reply:
POVRay is more complex to port, but I would be happy if someone is interested by this one

Reply:
I've successfully compiled POVRay for braswell, but currently it requires some tricks like without-openexr, and some manual changes in configure before automating the configure/make process.
But most of the hard work is completed for the cross part :)